### ISSUE TITLE
Wrong balance and missing send transaction 

### ISSUE THREAD

**Issue by blueish125** [bitpay/copay#6053]
Hi,  

Copay app is showing the wrong balance and a couple send transaction are missing.
Who do I need to talk to resolve this issue?

Blueish125

update 5/17/2017 12:02

Balance has corrected itself
still missing qty:2 send transactions

update 05/172017 10:57am
missing transactions reappeared
Issue resolved

**Comment by nrajesh** [comment]
See #6047. Auto resolved after several hours. But it was scary as hell for a normal person new to bitcoins!

**Comment by dricharson5150** [comment]
I'm still not seeing anything resolved still wrong balance and missing sent transaction that's sitting unconfirmed at an exchange!!!!

**Comment by dricharson5150** [comment]
12 hours now too!

**Comment by nrajesh** [comment]
I think the trouble is with the humongous backlog hogging bitcoin network. It's getting worse as per my quick understanding of this issue goes. But the moment confirmations arrive then the balance autocorrects in copay and even the disappeared transactions 'magically' appeared!

One option is to give a larger commission. Ironically that's what I did; this was the first time I chose highest commission fees instead of normal or economy that I usually give. Wallets like mycellium seem to have an option to raise the paid commission via another transaction.

Another option is to use a transaction accelerator service like the one viabtc offers (there are other paid one's as well). But I think it is best to wait for a day or two. For some folks their transaction now completes after 3-4 days!
